packages/fuel-indexer-macros/src/decoder.rs

```diff
@@ -673,14 +673,14 @@ impl From<ObjectDecoder> for TokenStream {
 
             impl From<#ident> for Json {
                 fn from(value: #ident) -> Self {
-                    let s = serde_json::to_string(&value).expect("Serde error.");
```

Comment:
- All of these errors were `"Serde error."` 
- Hard to sort through this in the repo, so I made them unique.